operation/operation.c

```diff
@@ -127,6 +127,7 @@ oper_st init_operation() {
     oper_st state;
 
     state.eeprom_log_idx = get_stored_value(LOG_INDEX_ADDR);
+    if(state.eeprom_log_idx >= MAX_ENTRIES) state.eeprom_log_idx = 0;
```

Comment:
add this to make sure if eeprom hasn't stored any log index then eeprom_log_idx  is within entries size